tests/Unit/TestHelpers.hpp

```diff
@@ -221,14 +222,30 @@ T serialize_and_deserialize(const T& t) {
 /// \ingroup TestingFrameworkGroup
 /// \brief Tests the serialization of comparable types
 /// \example
-/// To test serialization of comparable types use:
 /// \snippet Test_PupStlCpp11.cpp example_serialize_copyable
 template <typename T>
 void test_serialization(const T& t) {
   static_assert(tt::has_equivalence_v<T>, "No operator== for T");
   CHECK(t == serialize_and_deserialize(t));
 }
 
+/// \ingroup TestingFrameworkGroup
+/// \brief Test the serialization of a derived class via a base class pointer
+/// \example
+/// \snippet Test_PupStlCpp11.cpp example_serialize_derived
+template <typename B, typename D, typename... Args>
+void test_serialization_via_base(Args&&... args) {
```

Comment:
You should say what `args` is in the Doxygen comment. (arguments to the constructor of `D`)